lib/ansible/modules/messaging/rabbitmq_policy.py

```diff
@@ -51,13 +51,15 @@
     version_added: "2.1"
   pattern:
     description:
-      - A regex of queues to apply the policy to.
-    required: true
+      - A regex of queues to apply the policy to. Required when
+        C(state=present).
+    required: false
```

Comment:
this is changing the required/default from how the module used to work, you need to update the description and note that this changes in 2.4 vs the previous 'required' behaviour